node/node.go

```diff
@@ -329,46 +328,6 @@ func (n *Node) StartConsensus() error {
 	return nil
 }
 
-// CreateShardGenesisBlock creates the shard genesis block
-func (n *Node) CreateShardGenesisBlock() error {
-	header, err := genesis.CreateShardGenesisBlockFromInitialBalances(
-		n.accounts,
-		n.shardCoordinator,
-		n.addrConverter,
-		n.initialNodesBalances,
-		uint64(n.genesisTime.Unix()),
-	)
-	if err != nil {
-		return err
-	}
-
-	marshalizedHeader, err := n.marshalizer.Marshal(header)
-	if err != nil {
-		return err
-	}
-
-	blockHeaderHash := n.hasher.Compute(string(marshalizedHeader))
-
-	return n.setGenesis(header, blockHeaderHash)
-}
-
-// CreateMetaGenesisBlock creates the meta genesis block
-func (n *Node) CreateMetaGenesisBlock() error {
-	header, err := genesis.CreateMetaGenesisBlock(uint64(n.genesisTime.Unix()), n.initialNodesPubkeys)
```

Comment:
please add comment to core/genesis/genesisBlockCreator.go CreateMetaGenesisBlock func

Reply:
Done. Comment added.